Microsoft.Toolkit.HighPerformance/Box{T}.cs

```diff
@@ -83,15 +108,14 @@ public static Box<T> DangerousGetFrom(object obj)
         /// <param name="box">The resulting <see cref="Box{T}"/> reference, if <paramref name="obj"/> was a boxed <typeparamref name="T"/> value.</param>
         /// <returns><see langword="true"/> if a <see cref="Box{T}"/> instance was retrieved correctly, <see langword="false"/> otherwise.</returns>
         [MethodImpl(MethodImplOptions.AggressiveInlining)]
-        [SuppressMessage("StyleCop.CSharp.ReadabilityRules", "SA1115", Justification = "Comment for [NotNullWhen] attribute")]
         public static bool TryGetFrom(
             object obj,
-#if NETSTANDARD2_1
-            /* On .NET Standard 2.1, we can add the [NotNullWhen] attribute
-             * to let the code analysis engine know that whenever this method
-             * returns true, box will always be assigned to a non-null value.
-             * This will eliminate the null warnings when in a branch that
-             * is only taken when this method returns true. */
+#if NETSTANDARD2_1_OR_GREATER
+            // On .NET Standard 2.1, we can add the [NotNullWhen] attribute
+            // to let the code analysis engine know that whenever this method
+            // returns true, box will always be assigned to a non-null value.
+            // This will eliminate the null warnings when in a branch that
+            // is only taken when this method returns true.
             [NotNullWhen(true)]
```

Comment:
You should be able to also just define a shim class for `NETSTANDARD2_0` or less, which would relegate the condition to a single file.
(I believe the compiler will also respect it, etc)

Reply:
Ah, that's right, thanks! I think I didn't do this since that was the only case where I was using that attribute, but it makes perfect sense to put them in a separate file as you said. I'll mark them as `internal` as CoreCLR does so that they won't cause conflicts 👍